src/data/screenshots.json

```diff
@@ -9,7 +9,7 @@
         },
         "content": {
             "textblock": [
-                "Browse through the screenshots to get an overview of the features offered by the Corona-Warn-App with the release of Version 2.20 (available since ????? ??th, 2022). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
+                "Browse through the screenshots to get an overview of the features offered by the Corona-Warn-App with the release of Version 2.20 (available since March 20th, 2022). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
```

Comment:
This should probably be March 30th. 
March 20th is in the past anyway when there was not yet a version 2.20 released.

Reply:
We always have the same thoughts @MikeMcC399 (-;

Reply:
@Ein-Tim 
We worked in parallel again. When I started writing, there was nothing written from you! 😃